src/Geometries/VolumePrimitives/Tube.jl

```diff
@@ -56,20 +56,14 @@ function Geometry(T::DataType, t::Val{:tube}, dict::Dict{Union{Any,String}, Any}
 end
 
 function in(point::CartesianPoint{T}, tube::Tube{T}) where T
-    ismissing(tube.translate) ? nothing : point -= tube.translate
+    (ismissing(tube.translate) || tube.translate == CartesianVector{T}(0.0,0.0,0.0)) ? nothing : point -= tube.translate
```

Comment:
`tube.translate` should never be zero

Reply:
Well, we have to change the general struct of our primitives anyhow when we add Rotations. 
This `if ismissing` should never appear. I guess we will use parametric types to handle this on compilation level.

Reply:
For both, translation and rotation.